src/pages/SuperheroShow.js

```diff
@@ -2,11 +2,17 @@ import React, { Component } from 'react'
 
 class SuperheroShow extends Component {
     render () {
+        const hero = this.props.hero;
+        console.log(hero)
```

Comment:
try not to commit console.logs unless it's pertinent to the program. This seems like it's just for development/verifying purposes